packages/client/src/rpc/config.ts

```diff
@@ -0,0 +1,84 @@
+import type { MethodConfig } from '../util/rpc.ts'
+
+export const RPCNamespace = {
+  Eth: 'eth', // untrusted
+  Engine: 'engine', // trusted
+} as const
+
+export type RPCNamespace = (typeof RPCNamespace)[keyof typeof RPCNamespace]
+
+export const RPCTransport = {
```

Comment:
For the keys of this mapping and similar ones, we can switch to using lowercase key names, like [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/7d60a4f137ca3aa4370569ba277d4ad262d5d2c1/packages/client/src/logging.ts#L24).